npm/private/npm_link_package_store.bzl

```diff
@@ -76,6 +120,7 @@ def _impl(ctx):
     providers = [
         DefaultInfo(
             files = depset(files),
+            runfiles = ctx.runfiles(transitive_files),
```

Comment:
This was missing before? Is that related to this PR?

Reply:
It related to the sh_test added. It lets you pass a linked npm package to the data of a generic rule, such as sh_test, and get those files in the runfiles.

Reply:
That seems like a fix/feature on its own that could land separately? Or just it's own commit 🤷 

Is there an easy test we can add for this?

Reply:
I'll add a simple sh_test and move it to it own commit